.github/workflows/npm-test.yml

```diff
@@ -76,3 +76,26 @@ jobs:
       # Check that the storybook build succeeds
       - name: Build Storybook
         run: npm run build:storybook
+
+      # Install Playwright (with cache) for Storybook tests
+      - name: Get installed Playwright version
+        id: playwright-version
+        run: echo "PLAYWRIGHT_VERSION=$(node -e "console.log(require('./package-lock.json').dependencies['@playwright/test'].version)")" >> $GITHUB_ENV
+      - name: Cache playwright binaries
+        uses: actions/cache@v4
+        id: playwright-cache
+        with:
+          path: |
+            ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
+      - run: npx playwright install --with-deps
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+      - run: npx playwright install-deps
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+
+      # Run the storybook test suite
+      - name: Serve Storybook and run tests
+        run: |
+          npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
+            "npx http-server storybook-static --port 6006 --silent" \
+            "npx wait-on tcp:6006 && npx test-storybook"
```

Comment:
I see in CI that only chromium is used, let's enable firefox and webkit as well, see https://storybook.js.org/docs/writing-tests/test-runner#cli-options

Reply:
I've added more browsers 👍

Reply:
I've also reorganized scripts a bit, as I've noticed that running tets on my local machine worked only if I was already running storybook (`npm run storybook`) on separate terminal tab.

Now we have two scripts:
1. `test:storybook-ci` - assumes storybook is built, and assumes playwright is installed (both of these are already done in `.github/workflows/npm-test.yml`)
2. `test:storybook-locally` - builds storybook first (without it there is a risk that tests will be run on storybook built on different commit - I had that error), then installs playwright (without caching)

For running the tests, I've added `--testTimeout=30000 --maxWorkers=2` as the tests were failing sometimes on my machine.

Reply:
I also had to add some hack for webkit browser (linked the bug in comment)